src/sidebars/search/search-controller.js

```diff
@@ -19,15 +19,23 @@
             this._watch();
         }
 
+        hasStore() {
```

Comment:
I'll do `getStore`, as it allows reducing the boilerplate with setting up the store into one function :)